src/main/kotlin/pl/edu/agh/gem/external/persistence/NotVerifiedUserEntity.kt

```diff
@@ -0,0 +1,29 @@
+package pl.edu.agh.gem.external.persistence
+
+import org.springframework.data.annotation.Id
+import org.springframework.data.mongodb.core.index.Indexed
+import org.springframework.data.mongodb.core.mapping.Document
+import pl.edu.agh.gem.internal.model.auth.NotVerifiedUser
+import java.time.LocalDateTime
+
+@Document("not-verified")
+data class NotVerifiedUserEntity(
+    @Id
+    val id: String,
+    val email: String,
+    val password: String,
+    @Indexed(name = "ttl_index", expireAfter = "P30D")
```

Comment:
do we need to use that name, or sth else or auto name ?

Reply:
we don't. will remove it and then mongo will set default name